src/material-experimental/tree/tree.md

```diff
@@ -0,0 +1,91 @@
+The `mat-tree` provides a Material Design styled tree that can be used to display hierarchy
+data.
+
+This tree builds on the foundation of the CDK tree and uses a similar interface for its
+data source input and template, except that its element and attribute selectors will be prefixed
+with `mat-` instead of `cdk-`.
+
+There are two types of trees: Flat tree and Nested Tree. The DOM structures are different for these
```

Comment:
Small nitpick: _Flat **t**ree and Nested **t**ree_ or _Flat **T**ree and Nested **T**ree_